README.md

```diff
@@ -4,7 +4,7 @@
 
 <h2 align="center" style="margin-top:-10px">The Geospatial Neighborhood Analysis Package</h2> 
 
-[![Build Status](https://travis-ci.com/spatialucr/geosnap.svg?branch=master)](https://travis-ci.com/spatialucr/geosnap)
+![.github/workflows/unittests.yml](https://github.com/knaaptime/geosnap/workflows/.github/workflows/unittests.yml/badge.svg)
```

Comment:
When clicked, this badge only takes you to an image of the badge, not the test result itself. I had quite a time getting it correct in `spaghetti`. Here is what works for that (from [`README.md`](https://github.com/pysal/spaghetti/blob/master/README.md#spatial-graphs-networks-topology--inference)):

```
[![unittests](https://github.com/pysal/spaghetti/workflows/.github/workflows/unittests.yml/badge.svg)](https://github.com/pysal/spaghetti/actions?query=workflow%3A.github%2Fworkflows%2Funittests.yml)
```

Reply:
nice, thanks